common/qeidentity.c

```diff
@@ -58,17 +58,29 @@ oe_result_t oe_enforce_qe_identity(sgx_report_body_t* qe_report_body)
         // enclave's mrsigner.
         if (!oe_constant_time_mem_equal(
                 qe_report_body->mrsigner, g_qe_mrsigner, sizeof(g_qe_mrsigner)))
+        {
+            OE_TRACE_INFO("mrsigner mismatch\n");
```

Comment:
How about something like OE_ERROR("String", error_code), OE_ERROR then traces string and raises error code. It could also be called OE_TRACE_ERROR() or OE_TRACE_AND_RAISE()